.gitignore

```diff
@@ -0,0 +1,281 @@
+##Ignore test output files
+
+*.csv
+
```

Comment:
In the PR, this file is showing up as a new file, which should not be the case as there is one in the master brach.
One way to *probably* fix this  file then  merge master into your brach.